src/main/java/com/cburch/logisim/Main.java

```diff
@@ -52,27 +55,48 @@ public static void main(String[] args) {
       e.printStackTrace();
     }
 
-    final var startup = Startup.parseArgs(args);
-    if (startup == null) System.exit(10);
-    if (startup.shallQuit()) System.exit(0);
+    final var startup = new Startup(args);
+    final var loader = new Loader(null);
+    int exitCode = 0;
```

Comment:
Please replace all "magic values" (like `0`, `1`, `-1`) used as return code by `RC` enum values (see `Startup.java` etc. (multiple places). This (IMHO) should also affect `TtyInterface` code as we have these magic occurs there as well. Perhaps it should cleaner to separate `RC` from `Startup` class as well.

Reply:
RC enumerations are NOT the same as the program exit codes--there is not a one-to-one mapping. I will create exit codes symbols in Main and TtyInterface can use those (instead of defining them itself).